mswindows/osgeo4w/env.bat.tmpl

```diff
@@ -11,19 +11,21 @@ REM Note that msys package must be also installed
 REM set GRASS_SH=%OSGEO4W_ROOT%\apps\msys\bin\sh.exe
 
 set GRASS_PYTHON=%OSGEO4W_ROOT%\bin\python3.exe
-set PYTHONHOME=%OSGEO4W_ROOT%\apps\Python37
-
 set GRASS_PROJSHARE=%OSGEO4W_ROOT%\share\proj
 
+set FONTCONFIG_FILE=%GISBASE%\etc\fonts.conf
+
+REM
+REM GDAL-related
+REM
 set PROJ_LIB=%OSGEO4W_ROOT%\share\proj
 set GDAL_DATA=%OSGEO4W_ROOT%\share\gdal

```

Comment:
it'd defined in _C:\OSGeo4W64\etc\ini\gdal.bat_

`SET GDAL_DATA=%OSGEO4W_ROOT%\share\gdal`

also removing?

Reply:
Right, `gdal.bat` is called by `o4w_env.bat`. Removed by c2fd0e3c5